building adcprep v53release with gfortran on Ubuntu 20.04 fails

The build system seems to be looking for `icc`:
```
icc -I . -I /home/jason/work/adcirc-cg-v53release-gfortran/prep -O2 -DLINUX -I/home/jason/work/opt/default/include  -c /home/jason/work/adcirc-cg-v53release-gfortran/prep/mkdir.c -o /home/jason/work/adcirc-cg-v53release-gfortran/work/odir4/mkdir.o
/bin/bash: icc: command not found
make[1]: *** [makefile:443: /home/jason/work/adcirc-cg-v53release-gfortran/work/odir4/mkdir.o] Error 127
make[1]: Leaving directory '/home/jason/work/adcirc-cg-v53release-gfortran/work'
make: *** [makefile:480: padcirc] Error 2
build failed (2)
```

--- a/patches/ADCIRC/v53release-gfortran-desktop/01-v53release-qbc.patch
+++ b/patches/ADCIRC/v53release-gfortran-desktop/01-v53release-qbc.patch
@@ -1,0 +1,77 @@
+diff --git a/swan/macros.inc.queenbeeC b/swan/macros.inc.queenbeeC
+new file mode 100644
+index 0000000..6def14e
+--- /dev/null
++++ b/swan/macros.inc.queenbeeC
+@@ -0,0 +1,37 @@
++F90_SER = ifort
++F90_OMP = ifort
++F90_MPI = mpif90
++FLAGS_OPT = -O2
++FLAGS_MSC = -W0 -assume byterecl -traceback -diag-disable 8290 -diag-disable 8291 -diag-disable 8293 -assume buffered_io -xCORE-AVX512
++FLAGS90_MSC = $(FLAGS_MSC)
++FLAGS_SER =
++FLAGS_OMP = -openmp
++FLAGS_MPI =
++NETCDFROOT =
++ifneq ($(NETCDFROOT),)
++  INCS_SER = -I$(NETCDFROOT)/include
++  INCS_OMP = -I$(NETCDFROOT)/include
++  INCS_MPI = -I$(NETCDFROOT)/include
++  LIBS_SER = -L$(NETCDFROOT)/lib -lnetcdf -lnetcdff
++  LIBS_OMP = -L$(NETCDFROOT)/lib -lnetcdf -lnetcdff
++  LIBS_MPI = -L$(NETCDFROOT)/lib -lnetcdf -lnetcdff
++  NCF_OBJS = nctablemd.o agioncmd.o swn_outnc.o
++else
++  INCS_SER =
++  INCS_OMP =
++  INCS_MPI =
++  LIBS_SER =
++  LIBS_OMP =
++  LIBS_MPI =
++  NCF_OBJS =
++endif
++O_DIR = ../work/odir4/
++OUT = -o 
++EXTO = o
++MAKE = make
++RM = rm -f
++ifneq ($(NETCDFROOT),)
++  swch = -unix -impi -netcdf
++else
++  swch = -unix -impi
++endif
+diff --git a/work/cmplrflags.mk b/work/cmplrflags.mk
+index 2c0eb06..d95c766 100644
+--- a/work/cmplrflags.mk
++++ b/work/cmplrflags.mk
+@@ -277,6 +277,16 @@ ifeq ($(compiler),intel)
+         CFLAGS  := $(INCDIRS) -g -O0 -traceback -DLINUX -xSSE4.2 
+         FLIBS   := $(INCDIRS) -xSSE4.2 
+      endif
++  endif
++  ifeq ($(MACHINENAME),queenbeeC) 
++     FFLAGS1 := $(INCDIRS) -O3 -FI -assume byterecl -132 -xCORE-AVX512 -assume buffered_io
++     CFLAGS  := $(INCDIRS) -O3 -DLINUX -xCORE-AVX512 
++     FLIBS   := $(INCDIRS) -xCORE-AVX512 
++     ifeq ($(DEBUG),trace)
++        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xCORE-AVX512 -assume buffered_io
++        CFLAGS  := $(INCDIRS) -g -O0 -traceback -DLINUX -xCORE-AVX512 
++        FLIBS   := $(INCDIRS) -xCORE-AVX512 
++     endif
+   ifeq ($(MACHINENAME),supermic) 
+      FFLAGS1 := $(INCDIRS) -O3 -FI -assume byterecl -132 -xAVX -assume buffered_io
+      CFLAGS  := $(INCDIRS) -O3 -DLINUX -xAVX
+@@ -318,7 +328,11 @@ ifeq ($(compiler),intel)
+      # netcdf_fortran modules prior to compiling or executing ADCIRC
+      ifeq ($(MACHINENAME),queenbee)
+         FLIBS       := $(FLIBS) -L/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0/lib -lnetcdff -lnetcdf
+-        NETCDFHOME    :=/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0
++        NETCDFHOME  :=/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0
++     endif
++     ifeq ($(MACHINENAME),queenbeeC)
++        FLIBS       := $(FLIBS) -L/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0/lib -lnetcdff -lnetcdf
++        NETCDFHOME  :=/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0
+      endif
+      ifeq ($(MACHINENAME),supermic)
+         FLIBS      := $(FLIBS) -L /usr/local/packages/netcdf_fortran/4.2/INTEL-140-MVAPICH2-2.0/lib -lnetcdff -L/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0/lib -lnetcdf -lnetcdf -liomp5 -lpthread

--- a/patches/ADCIRC/v53release-gfortran-desktop/02-netcdfhome-cleanup.patch
+++ b/patches/ADCIRC/v53release-gfortran-desktop/02-netcdfhome-cleanup.patch
@@ -1,0 +1,80 @@
+diff --git a/work/cmplrflags.mk b/work/cmplrflags.mk
+index d95c766..141d448 100644
+--- a/work/cmplrflags.mk
++++ b/work/cmplrflags.mk
+@@ -90,12 +90,9 @@ ifeq ($(compiler),gnu)
+   ifeq ($(NETCDF),enable)
+      ifeq ($(MACHINENAME),blueridge)
+         # FLIBS       := $(FLIBS) -L$(HDF5HOME) -lhdf5  
+-        NETCDFHOME    :=/usr
+         FFLAGS1       :=$(FFLAGS1) -I/usr/lib64/gfortran/modules
+         FFLAGS2       :=$(FFLAGS1)
+         FFLAGS3       :=$(FFLAGS1)
+-        # NETCDFHOME  :=/shared/apps/RHEL-5/x86_64/NetCDF/netcdf-4.1.1-gcc4.1-ifort
+-        # NETCDFHOME  :=/shared/apps/RHEL-5/x86_64/NetCDF/netcdf-4.1.2-gcc4.1-ifort
+         FLIBS          :=$(FLIBS) -L/usr/lib64 -lnetcdff
+      else
+         FLIBS          := $(FLIBS) -L$(HDF5HOME) -lhdf5 -lhdf5_fortran
+@@ -318,49 +315,31 @@ ifeq ($(compiler),intel)
+   MSGLIBS       :=
+   ifeq ($(NETCDF),enable)
+      ifeq ($(MACHINENAME),hatteras)
+-        NETCDFHOME  :=/usr/share/Modules/software/CentOS-7/netcdf-Fortran/4.4.0_intel-18.0.0
+-        FLIBS       :=$(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+-        FFLAGS1     :=$(FFLAGS1) -I$(NETCDFHOME)/include
+-        FFLAGS2     :=$(FFLAGS1)
+-        FFLAGS3     :=$(FFLAGS1)
++        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -L$(NETCDFHOME)/lib -lnetcdf -lnetcdf
+      endif
+      # jgf20150417 queenbee requires that the analyst load the netcdf and
+      # netcdf_fortran modules prior to compiling or executing ADCIRC
+      ifeq ($(MACHINENAME),queenbee)
+         FLIBS       := $(FLIBS) -L/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0/lib -lnetcdff -lnetcdf
+-        NETCDFHOME  :=/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0
+      endif
+      ifeq ($(MACHINENAME),queenbeeC)
+         FLIBS       := $(FLIBS) -L/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0/lib -lnetcdff -lnetcdf
+-        NETCDFHOME  :=/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0
+      endif
+      ifeq ($(MACHINENAME),supermic)
+-        FLIBS      := $(FLIBS) -L /usr/local/packages/netcdf_fortran/4.2/INTEL-140-MVAPICH2-2.0/lib -lnetcdff -L/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0/lib -lnetcdf -lnetcdf -liomp5 -lpthread
+-        NETCDFHOME :=/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0/include
+-        FFLAGS1    :=$(FFLAGS1) -I/usr/local/packages/hdf5/1.8.12/INTEL-140-MVAPICH2-2.0/include
++        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -L$(NETCDFHOME)/lib -lnetcdf -lnetcdf
+      endif
+      ifeq ($(MACHINENAME),stampede)
+-        NETCDFHOME :=/opt/apps/intel17/netcdf/4.3.3.1/x86_64
+         FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+      endif
+      ifeq ($(MACHINENAME),stampede2)
+-        NETCDFHOME :=/opt/apps/intel17/netcdf/4.3.3.1/x86_64
+         FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+-        ifeq ($(USER),jgflemin)
+-           NETCDFHOME :=/work/00976/jgflemin/stampede2/local
+-           FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+-        endif
+      endif
+      ifeq ($(MACHINENAME),frontera)
+-        # specify NETCDFHOME on the command line or as an environment var
+         FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+      endif
+      # @jasonfleming: Added support for lonestar5 at tacc.utexas.edu;
+      # load the following module: netcdf/4.3.3.1
+      ifeq ($(MACHINENAME),lonestar5)
+-        #NETCDFHOME :=/opt/apps/intel18/netcdf/4.3.3.1/x86_64
+-        # @jasonfleming: Updated support for lonestar5
+-        NETCDFHOME :=/opt/apps/intel18/netcdf/4.6.2/x86_64
+         FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+      endif
+      # jgf20150817: Adding support for spirit.afrl.hpc.mil;
+@@ -373,10 +352,6 @@ ifeq ($(compiler),intel)
+      # jgf20150420 mike requires that the analyst add netcdf to the softenv
+      # with the following on the command line 
+      # soft add +netcdf-4.1.3-Intel-13.0.0
+-     ifeq ($(MACHINENAME),mike)
+-        FLIBS       := $(FLIBS) -L/usr/local/packages/netcdf/4.1.3/Intel-13.0.0/lib -lnetcdff -lnetcdf
+-        NETCDFHOME    :=/usr/local/packages/netcdf/4.1.3/Intel-13.0.0
+-     endif
+      ifeq ($(MACHINENAME),killdevil)
+         HDF5HOME       :=/nas02/apps/hdf5-1.8.5/lib
+         NETCDFHOME     :=/nas02/apps/netcdf-4.1.1

--- a/patches/ADCIRC/v53release-gfortran-desktop/03-issue-551.patch
+++ b/patches/ADCIRC/v53release-gfortran-desktop/03-issue-551.patch
@@ -1,0 +1,122 @@
+diff --git a/swan/macros.inc.queenbee b/swan/macros.inc.queenbee
+index e35a794..9fd6715 100644
+--- a/swan/macros.inc.queenbee
++++ b/swan/macros.inc.queenbee
+@@ -5,7 +5,7 @@ F90_SER = ifort
+ F90_OMP = ifort
+ F90_MPI = mpif90
+ FLAGS_OPT = -O2
+-FLAGS_MSC = -W0 -assume byterecl -traceback -diag-disable 8290 -diag-disable 8291 -diag-disable 8293 -assume buffered_io -xSSE4.2
++FLAGS_MSC = -W0 -traceback -diag-disable 8290 -diag-disable 8291 -diag-disable 8293 -xSSE4.2
+ FLAGS90_MSC = $(FLAGS_MSC)
+ FLAGS_SER =
+ FLAGS_OMP = -openmp
+diff --git a/swan/macros.inc.stampede2 b/swan/macros.inc.stampede2
+index 651ae8a..2f96d9b 100644
+--- a/swan/macros.inc.stampede2
++++ b/swan/macros.inc.stampede2
+@@ -5,7 +5,7 @@ F90_SER = ifort
+ F90_OMP = ifort
+ F90_MPI = mpif90
+ FLAGS_OPT = -O2
+-FLAGS_MSC = -W0 -assume byterecl -traceback -diag-disable 8290 -diag-disable 8291 -diag-disable 8293 -assume buffered_io -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512
++FLAGS_MSC = -W0 -traceback -diag-disable 8290 -diag-disable 8291 -diag-disable 8293 -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512
+ FLAGS90_MSC = $(FLAGS_MSC)
+ FLAGS_SER =
+ FLAGS_OMP = -openmp
+diff --git a/swan/macros.inc.supermic b/swan/macros.inc.supermic
+index 94d190e..71cc03b 100644
+--- a/swan/macros.inc.supermic
++++ b/swan/macros.inc.supermic
+@@ -2,7 +2,7 @@ F90_SER = ifort
+ F90_OMP = ifort
+ F90_MPI = mpif90
+ FLAGS_OPT = -O2
+-FLAGS_MSC = -W0 -assume byterecl -traceback -diag-disable 8290 -diag-disable 8291 -diag-disable 8293 -assume buffered_io -xAVX
++FLAGS_MSC = -W0 -traceback -diag-disable 8290 -diag-disable 8291 -diag-disable 8293 -xAVX
+ FLAGS90_MSC = $(FLAGS_MSC)
+ FLAGS_SER =
+ FLAGS_OMP = -openmp
+diff --git a/work/cmplrflags.mk b/work/cmplrflags.mk
+index 141d448..57c8cc7 100644
+--- a/work/cmplrflags.mk
++++ b/work/cmplrflags.mk
+@@ -246,8 +246,8 @@ ifeq ($(compiler),intel)
+   endif
+   #
+   ifeq ($(MACHINENAME),stampede2) 
+-     FFLAGS1 := $(INCDIRS) -O3 -FI -assume byterecl -132 -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512 -assume buffered_io
+-     CFLAGS  := $(INCDIRS) -O3 -DLINUX -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512 
++     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512 -assume buffered_io
++     CFLAGS  := $(INCDIRS) -O2 -DLINUX -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512 
+      FLIBS   := $(INCDIRS) -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512 
+      ifeq ($(DEBUG),trace)
+         FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512 -assume buffered_io
+@@ -256,8 +256,8 @@ ifeq ($(compiler),intel)
+      endif
+   endif
+   ifeq ($(MACHINENAME),frontera) 
+-     FFLAGS1 := $(INCDIRS) -O3 -FI -assume byterecl -132 -xCORE-AVX512 -assume buffered_io
+-     CFLAGS  := $(INCDIRS) -O3 -DLINUX -xCORE-AVX512 
++     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xCORE-AVX512 -assume buffered_io
++     CFLAGS  := $(INCDIRS) -O2 -DLINUX -xCORE-AVX512 
+      FLIBS   := $(INCDIRS) -xCORE-AVX512 
+      ifeq ($(DEBUG),trace)
+         FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xCORE-AVX512 -assume buffered_io
+@@ -266,8 +266,8 @@ ifeq ($(compiler),intel)
+      endif
+   endif
+   ifeq ($(MACHINENAME),queenbee) 
+-     FFLAGS1 := $(INCDIRS) -O3 -FI -assume byterecl -132 -xSSE4.2 -assume buffered_io
+-     CFLAGS  := $(INCDIRS) -O3 -DLINUX -xSSE4.2 
++     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xSSE4.2 -assume buffered_io
++     CFLAGS  := $(INCDIRS) -O2 -DLINUX -xSSE4.2 
+      FLIBS   := $(INCDIRS) -xSSE4.2 
+      ifeq ($(DEBUG),trace)
+         FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xSSE4.2 -assume buffered_io
+@@ -276,17 +276,18 @@ ifeq ($(compiler),intel)
+      endif
+   endif
+   ifeq ($(MACHINENAME),queenbeeC) 
+-     FFLAGS1 := $(INCDIRS) -O3 -FI -assume byterecl -132 -xCORE-AVX512 -assume buffered_io
+-     CFLAGS  := $(INCDIRS) -O3 -DLINUX -xCORE-AVX512 
++     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xCORE-AVX512 -assume buffered_io
++     CFLAGS  := $(INCDIRS) -O2 -DLINUX -xCORE-AVX512 
+      FLIBS   := $(INCDIRS) -xCORE-AVX512 
+      ifeq ($(DEBUG),trace)
+         FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xCORE-AVX512 -assume buffered_io
+         CFLAGS  := $(INCDIRS) -g -O0 -traceback -DLINUX -xCORE-AVX512 
+         FLIBS   := $(INCDIRS) -xCORE-AVX512 
+      endif
++  endif
+   ifeq ($(MACHINENAME),supermic) 
+-     FFLAGS1 := $(INCDIRS) -O3 -FI -assume byterecl -132 -xAVX -assume buffered_io
+-     CFLAGS  := $(INCDIRS) -O3 -DLINUX -xAVX
++     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xAVX -assume buffered_io
++     CFLAGS  := $(INCDIRS) -O2 -DLINUX -xAVX
+      FLIBS   := $(INCDIRS) -xAVX
+      ifeq ($(DEBUG),trace)
+         FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xAVX -assume buffered_io
+@@ -320,10 +321,10 @@ ifeq ($(compiler),intel)
+      # jgf20150417 queenbee requires that the analyst load the netcdf and
+      # netcdf_fortran modules prior to compiling or executing ADCIRC
+      ifeq ($(MACHINENAME),queenbee)
+-        FLIBS       := $(FLIBS) -L/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0/lib -lnetcdff -lnetcdf
++        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -L$(NETCDFHOME)/lib -lnetcdf
+      endif
+      ifeq ($(MACHINENAME),queenbeeC)
+-        FLIBS       := $(FLIBS) -L/usr/local/packages/netcdf/4.2.1.1/INTEL-140-MVAPICH2-2.0/lib -lnetcdff -lnetcdf
++        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -L$(NETCDFHOME)/lib -lnetcdf
+      endif
+      ifeq ($(MACHINENAME),supermic)
+         FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -L$(NETCDFHOME)/lib -lnetcdf -lnetcdf
+@@ -848,8 +849,7 @@ ifeq ($(compiler),circleci)
+      MULTIPLE := TRUE
+   endif
+ endif
+-#
+-endif
++
+ #$(MACHINE)
+ ########################################################################
+ # Compiler flags for Linux operating system on 32bit x86 CPU

--- a/patches/ADCIRC/v53release-gfortran-desktop/04-issue-549.patch
+++ b/patches/ADCIRC/v53release-gfortran-desktop/04-issue-549.patch
@@ -1,0 +1,163 @@
+diff --git a/swan/macros.inc.kraken b/swan/macros.inc.kraken
+index a8cf30b..7245699 100644
+--- a/swan/macros.inc.kraken
++++ b/swan/macros.inc.kraken
+@@ -7,9 +7,9 @@ F90_OMP = ftn
+ F90_MPI = ftn 
+ FLAGS_OPT =
+ FLAGS_MSC = 
+-FLAGS_SER = -O3 -static 
++FLAGS_SER = -O2 -static 
+ FLAGS_OMP = -mp
+-FLAGS_MPI = -O3 -static 
++FLAGS_MPI = -O2 -static 
+ INCS_SER =
+ INCS_OMP =
+ INCS_MPI =
+diff --git a/swan/platform.pl b/swan/platform.pl
+index 92842ed..57161bf 100644
+--- a/swan/platform.pl
++++ b/swan/platform.pl
+@@ -56,7 +56,7 @@ elsif ($os =~ /AIX/i) {
+   print OUTFILE "F90_SER = xlf90_r\n";
+   print OUTFILE "F90_OMP = xlf90_r\n";
+   print OUTFILE "F90_MPI = mpxlf90_r\n";
+-  print OUTFILE "FLAGS_OPT = -O3 -qstrict -qarch=auto -qtune=auto -qnohot -qcache=auto \\\n";
++  print OUTFILE "FLAGS_OPT = -O2 -qstrict -qarch=auto -qtune=auto -qnohot -qcache=auto \\\n";
+   print OUTFILE "            -qunroll -qalign=4k -qfloat=hsflt\n";
+   print OUTFILE "FLAGS_MSC = -w -qfixed -qnosave -q64\n";
+   print OUTFILE "FLAGS90_MSC = -w -qfree=f90 -qnosave -q64\n";
+@@ -491,7 +491,7 @@ elsif ($os =~ /Linux/i) {
+     print OUTFILE "F90_SER = xlf90\n";
+     print OUTFILE "F90_OMP =\n";
+     print OUTFILE "F90_MPI = mpfort\n";
+-    print OUTFILE "FLAGS_OPT = -O3 -qstrict -qarch=auto -qtune=auto\n";
++    print OUTFILE "FLAGS_OPT = -O2 -qstrict -qarch=auto -qtune=auto\n";
+     print OUTFILE "FLAGS_MSC = -qfixed -qzerosize -qwarn64\n";
+     print OUTFILE "FLAGS90_MSC = -qfree=f90 -qzerosize -qwarn64\n";
+     print OUTFILE "FLAGS_DYN =\n";
+@@ -636,7 +636,7 @@ elsif ($os =~ /Darwin/i) {
+   print OUTFILE "F90_SER = xlf90\n";
+   print OUTFILE "F90_OMP =\n";
+   print OUTFILE "F90_MPI =\n";
+-  print OUTFILE "FLAGS_OPT = -O3 -qstrict -qtune=auto -qcache=auto -qalign=4k\n";
++  print OUTFILE "FLAGS_OPT = -O2 -qstrict -qtune=auto -qcache=auto -qalign=4k\n";
+   print OUTFILE "FLAGS_MSC = -w -qfixed\n";
+   print OUTFILE "FLAGS90_MSC = -w -qfree=f90\n";
+   print OUTFILE "FLAGS_DYN =\n";
+diff --git a/work/cmplrflags.mk b/work/cmplrflags.mk
+index 57c8cc7..0ab9585 100644
+--- a/work/cmplrflags.mk
++++ b/work/cmplrflags.mk
+@@ -197,7 +197,7 @@ ifeq ($(compiler),g95)
+   PPFC		:=  g95
+   FC		:=  g95
+   PFC		:=  mpif90
+-  FFLAGS1	:=  $(INCDIRS) -O3 -mcmodel=medium -fstatic -ffixed-line-length-132
++  FFLAGS1	:=  $(INCDIRS) -O2 -mcmodel=medium -fstatic -ffixed-line-length-132
+   FFLAGS2	:=  $(FFLAGS1)
+   FFLAGS3	:=  $(FFLAGS1)
+   DA		:=  -DREAL8 -DLINUX -DCSCA
+@@ -374,7 +374,7 @@ ifeq ($(compiler),intel-ND)
+   PPFC            :=  ifort
+   FC            :=  ifort
+   PFC           :=  mpif90
+-  FFLAGS1       :=  $(INCDIRS) -w -O3 -assume byterecl -132 -i-dynamic -assume buffered_io
++  FFLAGS1       :=  $(INCDIRS) -w -O2 -assume byterecl -132 -i-dynamic -assume buffered_io
+   ifeq ($(DEBUG),full)
+      FFLAGS1       :=  $(INCDIRS) -g -O0 -traceback -debug -check all -i-dynamic -FI -assume byterecl -132 -DALL_TRACE -DFULL_STACK -DFLUSH_MESSAGES
+   endif
+@@ -389,7 +389,7 @@ ifeq ($(compiler),intel-ND)
+   IMODS         :=  -I
+   CC            := icc
+   CCBE          := $(CC)
+-  CFLAGS        := $(INCDIRS) -O3 -m64 -mcmodel=medium -DLINUX
++  CFLAGS        := $(INCDIRS) -O2 -m64 -mcmodel=medium -DLINUX
+   FLIBS          :=
+   ifeq ($(DEBUG),full)
+      CFLAGS        := $(INCDIRS) -g -O0 -march=k8 -m64 -mcmodel=medium -DLINUX
+@@ -706,8 +706,8 @@ ifeq ($(compiler),diamond)
+   PPFC          :=  ifort
+   FC            :=  ifort
+   PFC           :=  ifort
+-#  FFLAGS1       :=  $(INCDIRS) -O3 -xT -132
+-  FFLAGS1       := -O3 -132 -xSSSE3
++#  FFLAGS1       :=  $(INCDIRS) -O2 -xT -132
++  FFLAGS1       := -O2 -132 -xSSSE3
+   ifeq ($(DEBUG),full)
+      FFLAGS1	:=  $(INCDIRS) -g -O0 -debug -fpe0 -132 -traceback -check all -DALL_TRACE -DFLUSH_MESSAGES -DFULL_STACK
+   endif
+@@ -722,8 +722,8 @@ ifeq ($(compiler),diamond)
+   IMODS         :=  -I
+   CC            := icc
+   CCBE          := $(CC)
+-#  CFLAGS        := $(INCDIRS) -O3 -xT
+-  CFLAGS        := $(INCDIRS) -O3 -xSSSE3
++#  CFLAGS        := $(INCDIRS) -O2 -xT
++  CFLAGS        := $(INCDIRS) -O2 -xSSSE3
+   ifeq ($(DEBUG),full)
+      CFLAGS        := $(INCDIRS) -g -O0
+   endif
+@@ -799,7 +799,7 @@ ifeq ($(compiler),kraken)
+   PPFC          :=  ftn
+   FC            :=  ftn
+   PFC           :=  ftn
+-  FFLAGS1       :=  $(INCDIRS) -O3 -static  -132
++  FFLAGS1       :=  $(INCDIRS) -O2 -static  -132
+   FFLAGS2       :=  $(FFLAGS1)
+   FFLAGS3       :=  $(FFLAGS1)
+   DA            :=  -DREAL8 -DLINUX -DCSCA -DPOWELL
+@@ -1084,7 +1084,7 @@ ifeq ($(arch),altix)
+   PPFC            := ifort
+   FC              := ifort
+   PFC             := ifort
+-  FFLAGS1	  := $(INCDIRS) -O3 -tpp2
++  FFLAGS1	  := $(INCDIRS) -O2 -tpp2
+   FFLAGS2	  := $(FFLAGS1)
+   FFLAGS3	  := $(FFLAGS1)
+   DA	          :=  -DREAL8 -DCSCA
+@@ -1157,7 +1157,7 @@ ifeq ($(IBM),p5)
+   FFLAGS0       := $(INCDIRS) -w -qfixed=132 -qarch=auto -qcache=auto
+   FFLAGS1       := $(FFLAGS0) -O2
+   FFLAGS2       := $(FFLAGS0) -qhot -qstrict
+-  FFLAGS3       := $(FFLAGS0) -O3 -qinitauto
++  FFLAGS3       := $(FFLAGS0) -O2 -qinitauto
+   DA            := -WF,"-DREAL8,-DIBM,-DCSCA"
+   DP            := -tF -WF,"-DREAL8,-DIBM,-DCSCA,-DCMPI"
+   DPRE          := -tF -WF,"-DREAL8,-DIBM"
+@@ -1477,9 +1477,9 @@ ifneq (,$(findstring powerpc-darwin,$(MACHINE)-$(OS)))
+   PPFC	        := f90
+   FC	        := f90
+   PFC	        := mpif77
+-  FFLAGS1	:=  $(INCDIRS) -w -O3 -m64 -cpu:g5 -f fixed -W132 -I . -DLINUX
+-  FFLAGS2	:=  $(INCDIRS) -w -O3 -m64 -cpu:g5 -N11 -f fixed -W132 -I .
+-  FFLAGS3	:=  $(INCDIRS) -w -O3 -m64 -cpu:g5 -N11 -f fixed -W132 -I .
++  FFLAGS1	:=  $(INCDIRS) -w -O2 -m64 -cpu:g5 -f fixed -W132 -I . -DLINUX
++  FFLAGS2	:=  $(INCDIRS) -w -O2 -m64 -cpu:g5 -N11 -f fixed -W132 -I .
++  FFLAGS3	:=  $(INCDIRS) -w -O2 -m64 -cpu:g5 -N11 -f fixed -W132 -I .
+   DA  	   	:=  -DREAL8 -DCSCA -DLINUX
+   DP  	   	:=  -DREAL8 -DCSCA -DCMPI -DLINUX
+   DPRE	   	:=  -DREAL8 -DLINUX
+@@ -1507,17 +1507,17 @@ ifneq (,$(findstring i386-darwin,$(MACHINE)-$(OS)))
+   PPFC	        := ifort
+   FC	        := ifort
+   PFC	        := mpif77
+-  FFLAGS1       :=  $(INCDIRS) -nowarn -O3    -fixed -132 -check all -traceback -DLINUX -DNETCDF_DEBUG -I .
+-# FFLAGS1	:=  $(INCDIRS) -nowarn -O3    -fixed -132 -DIBM -I .
+-  FFLAGS2	:=  $(INCDIRS) -nowarn -O3    -fixed -132 -I .
+-  FFLAGS3	:=  $(INCDIRS) -nowarn -O3    -fixed -132 -I .
++  FFLAGS1       :=  $(INCDIRS) -nowarn -O2    -fixed -132 -check all -traceback -DLINUX -DNETCDF_DEBUG -I .
++# FFLAGS1	:=  $(INCDIRS) -nowarn -O2    -fixed -132 -DIBM -I .
++  FFLAGS2	:=  $(INCDIRS) -nowarn -O2    -fixed -132 -I .
++  FFLAGS3	:=  $(INCDIRS) -nowarn -O2    -fixed -132 -I .
+   DA  	   	:=  -DREAL8 -DCSCA -DLINUX
+   DP  	   	:=  -DREAL8 -DCSCA -DLINUX -DCMPI -DNETCDF_DEBUG
+   DPRE	   	:=  -DREAL8 -DLINUX
+   IMODS  	:=  -I
+   CC            :=  gcc
+   CCBE          :=  $(CC)
+-  CFLAGS        :=  $(INCDIRS) -O3 -DLINUX
++  CFLAGS        :=  $(INCDIRS) -O2 -DLINUX
+   LDFLAGS	:=
+   FLIBS	        :=
+   MSGLIBS	:=

--- a/patches/ADCIRC/v53release-gfortran-desktop/05-issue-550.patch
+++ b/patches/ADCIRC/v53release-gfortran-desktop/05-issue-550.patch
@@ -1,0 +1,102 @@
+diff --git a/work/cmplrflags.mk b/work/cmplrflags.mk
+index 0ab9585..8b19b62 100644
+--- a/work/cmplrflags.mk
++++ b/work/cmplrflags.mk
+@@ -226,7 +226,7 @@ ifeq ($(compiler),intel)
+   PPFC            :=  ifort
+   FC            :=  ifort
+   PFC           :=  mpif90
+-  FFLAGS1       :=  $(INCDIRS) -O2 -FI -assume byterecl -132 -xSSE4.2 -assume buffered_io
++  FFLAGS1       :=  $(INCDIRS) -O2 -FI -assume byterecl -132 -xSSE4.2
+   CFLAGS        := $(INCDIRS) -O2 -xSSE4.2 -m64 -mcmodel=medium -DLINUX
+   FLIBS         :=
+   ifeq ($(DEBUG),full)
+@@ -246,51 +246,51 @@ ifeq ($(compiler),intel)
+   endif
+   #
+   ifeq ($(MACHINENAME),stampede2) 
+-     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512 -assume buffered_io
++     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512
+      CFLAGS  := $(INCDIRS) -O2 -DLINUX -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512 
+      FLIBS   := $(INCDIRS) -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512 
+      ifeq ($(DEBUG),trace)
+-        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512 -assume buffered_io
++        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512
+         CFLAGS  := $(INCDIRS) -g -O0 -traceback -DLINUX -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512 
+         FLIBS   := $(INCDIRS) -xCORE-AVX2 -axCORE-AVX512,MIC-AVX512 
+      endif
+   endif
+   ifeq ($(MACHINENAME),frontera) 
+-     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xCORE-AVX512 -assume buffered_io
++     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xCORE-AVX512
+      CFLAGS  := $(INCDIRS) -O2 -DLINUX -xCORE-AVX512 
+      FLIBS   := $(INCDIRS) -xCORE-AVX512 
+      ifeq ($(DEBUG),trace)
+-        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xCORE-AVX512 -assume buffered_io
++        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xCORE-AVX512
+         CFLAGS  := $(INCDIRS) -g -O0 -traceback -DLINUX -xCORE-AVX512
+         FLIBS   := $(INCDIRS) -xCORE-AVX512 
+      endif
+   endif
+   ifeq ($(MACHINENAME),queenbee) 
+-     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xSSE4.2 -assume buffered_io
++     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xSSE4.2
+      CFLAGS  := $(INCDIRS) -O2 -DLINUX -xSSE4.2 
+      FLIBS   := $(INCDIRS) -xSSE4.2 
+      ifeq ($(DEBUG),trace)
+-        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xSSE4.2 -assume buffered_io
++        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xSSE4.2
+         CFLAGS  := $(INCDIRS) -g -O0 -traceback -DLINUX -xSSE4.2 
+         FLIBS   := $(INCDIRS) -xSSE4.2 
+      endif
+   endif
+   ifeq ($(MACHINENAME),queenbeeC) 
+-     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xCORE-AVX512 -assume buffered_io
++     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xCORE-AVX512
+      CFLAGS  := $(INCDIRS) -O2 -DLINUX -xCORE-AVX512 
+      FLIBS   := $(INCDIRS) -xCORE-AVX512 
+      ifeq ($(DEBUG),trace)
+-        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xCORE-AVX512 -assume buffered_io
++        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xCORE-AVX512
+         CFLAGS  := $(INCDIRS) -g -O0 -traceback -DLINUX -xCORE-AVX512 
+         FLIBS   := $(INCDIRS) -xCORE-AVX512 
+      endif
+   endif
+   ifeq ($(MACHINENAME),supermic) 
+-     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xAVX -assume buffered_io
++     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xAVX
+      CFLAGS  := $(INCDIRS) -O2 -DLINUX -xAVX
+      FLIBS   := $(INCDIRS) -xAVX
+      ifeq ($(DEBUG),trace)
+-        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xAVX -assume buffered_io
++        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xAVX
+         CFLAGS  := $(INCDIRS) -g -O0 -traceback -DLINUX  -xAVX 
+         FLIBS   := $(INCDIRS) -xAVX
+      endif
+@@ -374,7 +374,7 @@ ifeq ($(compiler),intel-ND)
+   PPFC            :=  ifort
+   FC            :=  ifort
+   PFC           :=  mpif90
+-  FFLAGS1       :=  $(INCDIRS) -w -O2 -assume byterecl -132 -i-dynamic -assume buffered_io
++  FFLAGS1       :=  $(INCDIRS) -w -O2 -assume byterecl -132 -i-dynamic
+   ifeq ($(DEBUG),full)
+      FFLAGS1       :=  $(INCDIRS) -g -O0 -traceback -debug -check all -i-dynamic -FI -assume byterecl -132 -DALL_TRACE -DFULL_STACK -DFLUSH_MESSAGES
+   endif
+@@ -418,7 +418,7 @@ ifeq ($(compiler),intel-sgi)
+   PFC           :=  mpif90
+   CC            :=  icc -O2 -no-ipo
+   CCBE          :=  icc -O2 -no-ipo
+-  FFLAGS1       :=  $(INCDIRS) -fixed -extend-source 132 -O2 -finline-limit=1000 -real-size 64 -no-ipo -assume buffered_io
++  FFLAGS1       :=  $(INCDIRS) -fixed -extend-source 132 -O2 -finline-limit=1000 -real-size 64 -no-ipo
+ #  FFLAGS1      :=  $(INCDIRS) -Mextend -g -O0 -traceback
+   FFLAGS2       :=  $(FFLAGS1)
+   FFLAGS3       :=  $(FFLAGS1) -assume buffered_stdout
+@@ -574,7 +574,7 @@ ifeq ($(compiler),xtintel)
+   PFC           :=  ftn
+   CC            :=  cc -O2 -no-ipo
+   CCBE          :=  cc -O2 -no-ipo
+-  FFLAGS1       :=  $(INCDIRS) -fixed -extend-source 132 -O2 -default64 -finline-limit=1000 -real-size 64 -no-ipo -assume buffered_io
++  FFLAGS1       :=  $(INCDIRS) -fixed -extend-source 132 -O2 -default64 -finline-limit=1000 -real-size 64 -no-ipo
+ #  FFLAGS1      :=  $(INCDIRS) -Mextend -g -O0 -traceback
+   FFLAGS2       :=  $(FFLAGS1)
+   FFLAGS3       :=  $(FFLAGS1) -assume buffered_stdout

--- a/patches/ADCIRC/v53release-gfortran-desktop/06-issue-743-rostam.patch
+++ b/patches/ADCIRC/v53release-gfortran-desktop/06-issue-743-rostam.patch
@@ -1,0 +1,31 @@
+diff --git a/work/cmplrflags.mk b/work/cmplrflags.mk
+index 8b19b62..23fefb9 100644
+--- a/work/cmplrflags.mk
++++ b/work/cmplrflags.mk
+@@ -295,6 +295,16 @@ ifeq ($(compiler),intel)
+         FLIBS   := $(INCDIRS) -xAVX
+      endif
+   endif
++  ifeq ($(MACHINENAME),rostam) 
++     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xAVX
++     CFLAGS  := $(INCDIRS) -O2 -DLINUX -xAVX
++     FLIBS   := $(INCDIRS) -xAVX
++     ifeq ($(DEBUG),trace)
++        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xAVX
++        CFLAGS  := $(INCDIRS) -g -O0 -traceback -DLINUX  -xAVX 
++        FLIBS   := $(INCDIRS) -xAVX
++     endif
++  endif
+   endif
+   #
+   #@jasonfleming Added to fix bus error on hatteras@renci
+@@ -329,6 +339,9 @@ ifeq ($(compiler),intel)
+      ifeq ($(MACHINENAME),supermic)
+         FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -L$(NETCDFHOME)/lib -lnetcdf -lnetcdf
+      endif
++     ifeq ($(MACHINENAME),rostam)
++        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -L$(NETCDFHOME)/lib -lnetcdf -lnetcdf
++     endif
+      ifeq ($(MACHINENAME),stampede)
+         FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+      endif

--- a/patches/ADCIRC/v53release-gfortran-desktop/07-issue-820-lonestar6.patch
+++ b/patches/ADCIRC/v53release-gfortran-desktop/07-issue-820-lonestar6.patch
@@ -1,0 +1,14 @@
+diff --git a/work/cmplrflags.mk b/work/cmplrflags.mk
+index 23fefb9..511117b 100644
+--- a/work/cmplrflags.mk
++++ b/work/cmplrflags.mk
+@@ -356,6 +356,9 @@ ifeq ($(compiler),intel)
+      ifeq ($(MACHINENAME),lonestar5)
+         FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+      endif
++     ifeq ($(MACHINENAME),ls6)
++        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
++     endif
+      # jgf20150817: Adding support for spirit.afrl.hpc.mil;
+      # load the following modules: netcdf-fortran/intel/4.4.2
+      # and hdf5/intel/1.8.12 and hdf5-mpi/intel/sgimpt/1.8.12

--- a/patches/ADCIRC/v53release-gfortran-desktop/08-issue-820-lonestar6-2.patch
+++ b/patches/ADCIRC/v53release-gfortran-desktop/08-issue-820-lonestar6-2.patch
@@ -1,0 +1,21 @@
+diff --git a/work/cmplrflags.mk b/work/cmplrflags.mk
+index 511117b..ab80a66 100644
+--- a/work/cmplrflags.mk
++++ b/work/cmplrflags.mk
+@@ -305,6 +305,16 @@ ifeq ($(compiler),intel)
+         FLIBS   := $(INCDIRS) -xAVX
+      endif
+   endif
++  ifeq ($(MACHINENAME),ls6) 
++     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132
++     CFLAGS  := $(INCDIRS) -O2 -DLINUX
++     FLIBS   := $(INCDIRS)
++     ifeq ($(DEBUG),trace)
++        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132
++        CFLAGS  := $(INCDIRS) -g -O0 -traceback -DLINUX
++        FLIBS   := $(INCDIRS)
++     endif
++  endif
+   endif
+   #
+   #@jasonfleming Added to fix bus error on hatteras@renci

--- a/patches/ADCIRC/v53release-gfortran-desktop/09-issue-840-supermike-iii.patch
+++ b/patches/ADCIRC/v53release-gfortran-desktop/09-issue-840-supermike-iii.patch
@@ -1,0 +1,62 @@
+diff --git a/work/cmplrflags.mk b/work/cmplrflags.mk
+index ab80a66..2fc5063 100644
+--- a/work/cmplrflags.mk
++++ b/work/cmplrflags.mk
+@@ -315,6 +315,20 @@ ifeq ($(compiler),intel)
+         FLIBS   := $(INCDIRS)
+      endif
+   endif
++  #wwlwpd: NOTE: Mike III at LSU-HPC recommends 'mpiifort', Intel's MPI implementation wrapper
++  #wwlwpd: instead of the traditional "mpif90"; but the flags are the same we still redefine
++  #wwlwpd: "PFC" below
++  ifeq ($(MACHINENAME),mike)
++     PFC     :=  mpiifort
++     FFLAGS1 := $(INCDIRS) -O2 -FI -assume byterecl -132 -xAVX
++     CFLAGS  := $(INCDIRS) -O2 -DLINUX -xAVX
++     FLIBS   := $(INCDIRS) -xAVX
++     ifeq ($(DEBUG),trace)
++        FFLAGS1 := $(INCDIRS) -g -O0 -traceback -FI -assume byterecl -132 -xAVX
++        CFLAGS  := $(INCDIRS) -g -O0 -traceback -DLINUX  -xAVX
++        FLIBS   := $(INCDIRS) -xAVX
++     endif
++  endif
+   endif
+   #
+   #@jasonfleming Added to fix bus error on hatteras@renci
+@@ -336,21 +350,21 @@ ifeq ($(compiler),intel)
+   MSGLIBS       :=
+   ifeq ($(NETCDF),enable)
+      ifeq ($(MACHINENAME),hatteras)
+-        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -L$(NETCDFHOME)/lib -lnetcdf -lnetcdf
++        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+      endif
+      # jgf20150417 queenbee requires that the analyst load the netcdf and
+      # netcdf_fortran modules prior to compiling or executing ADCIRC
+      ifeq ($(MACHINENAME),queenbee)
+-        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -L$(NETCDFHOME)/lib -lnetcdf
++        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+      endif
+      ifeq ($(MACHINENAME),queenbeeC)
+-        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -L$(NETCDFHOME)/lib -lnetcdf
++        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+      endif
+      ifeq ($(MACHINENAME),supermic)
+-        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -L$(NETCDFHOME)/lib -lnetcdf -lnetcdf
++        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+      endif
+      ifeq ($(MACHINENAME),rostam)
+-        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -L$(NETCDFHOME)/lib -lnetcdf -lnetcdf
++        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+      endif
+      ifeq ($(MACHINENAME),stampede)
+         FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+@@ -369,6 +383,9 @@ ifeq ($(compiler),intel)
+      ifeq ($(MACHINENAME),ls6)
+         FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+      endif
++     ifeq ($(MACHINENAME),mike)
++        FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
++     endif
+      # jgf20150817: Adding support for spirit.afrl.hpc.mil;
+      # load the following modules: netcdf-fortran/intel/4.4.2
+      # and hdf5/intel/1.8.12 and hdf5-mpi/intel/sgimpt/1.8.12

--- a/patches/ADCIRC/v53release-gfortran-desktop/10-issue-819-gfortran-desktop.patch
+++ b/patches/ADCIRC/v53release-gfortran-desktop/10-issue-819-gfortran-desktop.patch
@@ -1,0 +1,33 @@
+--- adcirc-cg-v53release-gfortran-desktop/work/cmplrflags.mk	2022-07-20 12:04:20.813959160 -0400
++++ adcirc-cg-v53release-gfortran-9-desktop/work/cmplrflags.mk	2022-07-20 12:12:23.146735838 -0400
+@@ -162,11 +162,8 @@
+         FFLAGS3 := $(FFLAGS3) -I${HOME}/local/include
+         FLIBS := $(FLIBS) -L${HOME}/local/lib -lnetcdff -lnetcdf
+      endif 
+-     ifeq ($(MACHINENAME),jason-desktop)
+-        NETCDFHOME := /usr
+-        FFLAGS1 := $(FFLAGS1) -L/usr/lib/x86_64-linux-gnu
+-        FFLAGS2 := $(FFLAGS2) -L/usr/lib/x86_64-linux-gnu
+-        FFLAGS3 := $(FFLAGS3) -L/usr/lib/x86_64-linux-gnu
++     ifeq ($(MACHINENAME),desktop)
++	FLIBS      := $(FLIBS) -L$(NETCDFHOME)/lib -lnetcdff -lnetcdf
+      endif
+      ifeq ($(MACHINENAME),rostam)    
+         NETCDFHOME := /usr
+@@ -329,7 +326,6 @@
+         FLIBS   := $(INCDIRS) -xAVX
+      endif
+   endif
+-  endif
+   #
+   #@jasonfleming Added to fix bus error on hatteras@renci
+   ifeq ($(HEAP_ARRAYS),fix)
+@@ -892,7 +888,7 @@
+      MULTIPLE := TRUE
+   endif
+ endif
+-
++endif
+ #$(MACHINE)
+ ########################################################################
+ # Compiler flags for Linux operating system on 32bit x86 CPU

--- a/patches/ADCIRC/v53release-gfortran-desktop/about.txt
+++ b/patches/ADCIRC/v53release-gfortran-desktop/about.txt
@@ -1,0 +1,1 @@
+v53 for gfortran 9 on desktop

--- a/patches/ADCIRC/v53release-gfortran-desktop/info.sh
+++ b/patches/ADCIRC/v53release-gfortran-desktop/info.sh
@@ -1,0 +1,3 @@
+export ADCIRC_GIT_BRANCH=v53release
+export SWANDIR=swan
+export PATCHSET_NAME=

